content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -917,7 +917,7 @@ value: ../../../
  src/PimEnterprise/Bundle/InstallerBundle/Resources/fixtures/icecat_demo/images/AKNTS_PB.jpg
 ```
 
-We expect a **text** as `value`.
+We expect a **text** as `value` (it corresponds to the filename).
```

Comment:
`corresponds` seems weird to me, event if this word exists in english. I would use `it should match the filename`.  
Maybe ask to another person :smile:

Reply:
@camillebourjade what do you think about "corresponds"? Thanks :)

Reply:
I agree with @nmarniesse :-) 
The 'expect' should  be 'require'